src/qt/bitcoin.cpp

```diff
@@ -381,6 +378,16 @@ void BitcoinApplication::requestShutdown()
 void BitcoinApplication::initializeResult(bool success, interfaces::BlockAndHeaderTipInfo tip_info)
 {
     qDebug() << __func__ << ": Initialization result: " << success;
+
+    if (m_splash) {
+        // If success, m_splash is no longer needed.
```

Comment:
0370fee74b2d665851214f15c0ba2d28e544a40b#

according to the comment here, the condition above should be `if (success)` ?

Reply:
The `if (m_splash)` condition actually guards the case when `bitcoin-qt` is run with `-nosplash`.

In the comment, I was going to emphasize on the fact the we are going to delete `m_splash` _regardless_ of `success`.
But motivation of such deletion depends on `success`.

Reply:
Unneeded condition has been removed.